### PR TITLE
[FIX] web: image field doesn't correctly update its unique

### PR DESCRIPTION
**Pull request by FrancoisGe**
Before this commit, if the value of an image field is modified by an onchange and the record is saved manually, the old image is displayed.

Why:
ImageField's rawCacheKey, which is used to know if the image has been updated, is not updated. So we will reuse the old image.

Solution:
rawCacheKey becomes a getter that always returns the current value of __last_updated.

How to reproduce:
- Go to a form view with a char field and an image field
- Edit the char field
- An onchange is performed and modifies the value of the image field
- The new image is displayed
- Click on the save button

Before this commit:
    The old image is displayed

After this commit:
    The new image is displayed

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/views/fields/image/image_field.js [hunk 1]
```diff
@@ -105,8 +97,6 @@ export class ImageField extends Component {
     }
     onFileUploaded(info) {
         this.state.isValid = true;
-        // Invalidate the `rawCacheKey`.
-        this.rawCacheKey = null;
```
Comment (aab-odoo) [inline addons/web/static/src/views/fields/image/image_field.js:109]: what if there's no onchange that modifies the last_update field when a new image is uploaded? How is the new image correctly displayed in this case?

File: addons/web/static/tests/views/fields/image_field_tests.js [hunk 2]
```diff
@@ -366,6 +366,58 @@ QUnit.module("Fields", (hooks) => {
         }
     );
 
+    QUnit.test(
+        "clicking save manually after onchange has been changed the image value should change the unique of the image src",
```
Comment (aab-odoo) [inline addons/web/static/tests/views/fields/image_field_tests.js:370]:
```suggestion
        "save record with image field modified by onchange",
```

File: addons/web/static/src/views/fields/image/image_field.js [hunk 3]
```diff
@@ -129,6 +124,7 @@ ImageField.props = {
     acceptedFileExtensions: { type: String, optional: true },
     width: { type: Number, optional: true },
     height: { type: Number, optional: true },
+    noReload: { type: Boolean, optional: true },
```
[the remark this answers is not in the document]
Comment (FrancoisGe) [reply addons/web/static/src/views/fields/image/image_field.js:127]: I agree but it s back port from v17

**Review by aab-odoo — APPROVED** [review]
robodoo rebase-ff r+

**Pull request closed without merging**